### PR TITLE
Transformed MCI changes

### PR DESCRIPTION
**Pull request by nealedj**
### What is the context of this PR?
Content changes to transformed MCI survey prototype as per slides.

### How to review 
Launch survey and compare against slides.

### CODE REVIEW

File: data/en/mci_transformation.json [hunk 1]
```diff
@@ -524,15 +524,15 @@
                     "title": "Retail sales including VAT",
                     "questions": [{
                         "id": "food-sales-question",
-                        "title": "Of the {{answers['retail-sales-answer']|format_currency}} retail sales, how much were your <em>food sales</em>?",
+                        "title": "Of the {{answers['retail-sales-answer']|format_currency}} retail sales, how much were your <em>food sales, drink and tobacco sales</em>?",
```
Comment (dwyeradam) [inline data/en/mci_transformation.json]: Should be just food. Not food `sales`.

File: data/en/mci_transformation.json [hunk 2]
```diff
@@ -524,15 +524,15 @@
...
                         "guidance": {
                             "content": [{
                                     "title": "Include",
                                     "list": [
-                                        "all fresh food",
-                                        "other food for human consumption (except chocolate and sugar confectionery)",
-                                        "soft drinks"
+                                        "all food for human consumption",
+                                        "alcoholic and non-alcoholic drinks",
+                                        "tobacco products and smokers’ requisites (including e-cigarettes)"
```
Comment (dwyeradam) [inline data/en/mci_transformation.json:526]: These changes must also be matched on the Introduction page's `Information you need`. See slide 4.

**Review by kierenbeckett — APPROVED** [review]
Besides Adam's comments LGTM

**Review by dwyeradam — CHANGES_REQUESTED** [review]
The include/exclude changes must also be matched on the Introduction page's Information you need. See slide 4.

**Comment by nealedj** [comment]
Thanks @dwyeradam - ready for re-review

**Comment by dwyeradam** [comment]
@nealedj, i'll review now. I'm guessing it's correct so do you want to squash now and save 20 mins.

File: data/en/mci_transformation.json [hunk 3]
```diff
@@ -100,75 +100,62 @@
...
                                     "list": [
-                                        "clothing and footwear",
-                                        "clothing fabrics",
-                                        "haberdashery and furs",
-                                        "leather and travel goods",
-                                        "handbags and umbrellas"
+                                        "all clothing and footwear",
+                                        "clothing fabrics and haberdashery",
+                                        "accessories such as belts, handbags and umbrellas",
+                                        "travel cases and luggage"
```
Comment (dwyeradam) [inline data/en/mci_transformation.json]: Talked too soon. Missing `Exclude` list from Clothing and footwear sales.

**Review by dwyeradam — DISMISSED** [review]
Missed one. Also just my OCD but it should be a lower case T for Turnover in Total turnover on the Information you need. I wouldn't have said if it wasn't for the other change. I also think it should be lower case S for Retail Sales but inconsistent.

**Review by dwyeradam — CHANGES_REQUESTED** [review]
See above.

**Review by MebinAbraham — COMMENTED** [review]
Even though it isn't mentioned on the card, to match the slides, `retail sales` should have `em` tags for the following. More of a while you are here...

https://github.com/ONSdigital/eq-survey-runner/blob/d1be5e9c8df7daad5b619c106124b2e467adeffc/data/en/mci_transformation.json#L760

https://github.com/ONSdigital/eq-survey-runner/blob/d1be5e9c8df7daad5b619c106124b2e467adeffc/data/en/mci_transformation.json#L814

https://github.com/ONSdigital/eq-survey-runner/blob/d1be5e9c8df7daad5b619c106124b2e467adeffc/data/en/mci_transformation.json#L865

**Comment by nealedj** [comment]
@dwyeradam @MebinAbraham good catches. Should all be addressed now.

**Comment by MebinAbraham** [comment]
@nealedj Needs a `yarn format`

**Review by dwyeradam — APPROVED** [review]
Ready to squash and merge. I think you now need to add a line for linting.

**Review by MebinAbraham — APPROVED** [review]
👍

**Pull request merged**